python/lsst/ts/observatory/control/maintel/mtcs.py

```diff
@@ -1401,6 +1401,68 @@ async def is_m1m3_in_engineering_mode(self) -> bool:
 
         return m1m3_detailed_state in self.m1m3_engineering_states
 
+    async def run_m2_actuator_bump_test(
+        self,
+        actuator: int,
+        period: float,
+        force: float,
+    ) -> None:
+        """M2 actuator bump test.
+
+        Parameters
+        ----------
+        actuator : `int`
+            Id of the actuator.
+        period : `float`,
+            There will be two bumps and each bump will wait for (2 * period)
+            seconds.
+        force : `float`, optional
+            the +/- push/pull foce to be applied in N
+        """
+        # check that actuator_id is not hardpoint
+        hardpoint_ids = await self.get_m2_hardpoints()
+
+        # csc actuator id is 0 based and hardpoint id is 1 based
+        if actuator + 1 in hardpoint_ids:
+            raise Exception(
+                f"Cannot bump test one of the M2 hardpoints: actuator = {actuator}"
+            )
+
+        # bump test single actuator
+        await self.rem.mtm2.cmd_actuatorBumpTest.set_start(
+            actuator=actuator,
+            period=period,
+            force=force,
+        )
+
+    async def get_m2_hardpoints(
+        self,
+    ) -> list:
```

Comment:
I think you can specify that this is a list of ints, something like `list[int]`, otherwise mypy assumes it is a list of anything.